repos/fdbt-netex-output/src/netex-convertor/netexGenerator.ts

```diff
@@ -352,6 +354,12 @@ const netexGenerator = async (ticket: Ticket, operatorData: Operator[]): Promise
     const updatePriceFareFrame = (priceFareFrame: NetexObject): NetexObject => {
         const priceFareFrameToUpdate = { ...priceFareFrame };
 
```

Comment:
const isPricedByDistance = 'pricingByDistance' in ticket.products[0];

Reply:
Ah you mean move the boolean to a variable